### PR TITLE
add directive lists in document

### PR DESCRIPTION
**Pull request by i110**
Before
before

After

after

### CODE REVIEW

File: doc/Makefile [hunk 1]
```diff
@@ -1,4 +1,6 @@
-BINARY_DIR=..
+ifndef BINARY_DIR
+	BINARY_DIR:=..
+endif
```
Comment (kazuho) [inline doc/Makefile]: Wy do you need this change?

My understanding is that without this change you can override the value by `make BINARY_DIR=foo`.

Comment (i110) [reply doc/Makefile]: oh, I didn't know that! will remove it.

Comment (i110) [reply doc/Makefile]: @kazuho もとに戻しました at https://github.com/h2o/h2o/pull/1547/commits/1d6e644de2d2edf7e385e743d8cce0ec0f5d54c3

**Comment by kazuho** [comment]
Thank you for the changes!

**Pull request merged**